platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/IssDetectorWiring.java

```diff
@@ -48,12 +53,24 @@ public record IssDetectorWiring(
      */
     @NonNull
     public static IssDetectorWiring create(@NonNull final TaskScheduler<List<IssNotification>> taskScheduler) {
+        final OutputWire<IssNotification> notificationOutputWire =
+                taskScheduler.getOutputWire().buildSplitter("issNotificationSplitter", "iss notifications");
+        final OutputWire<PlatformStatusAction> catastrophicFailureWire =
+                notificationOutputWire.buildTransformer("toStatusAction", "notification", notification -> {
+                    if (Set.of(IssNotification.IssType.SELF_ISS, IssNotification.IssType.CATASTROPHIC_ISS)
+                            .contains(notification.getIssType())) {
+                        return new CatastrophicFailureAction();
+                    }
+                    // don't change status for other types of ISS
+                    return null;
+                });
```

Comment:
When the ISS detector properly uses proxy wiring this should be a method on the component. But ok for it to live here until then.